architecture-model-generator/modules/architecture-model-generator-ls-extension/src/test/resources/results/frontend_service.json

```diff
@@ -748,20 +748,20 @@
               "elementLocation": {
                 "filePath": "{srcPath}/ballerina/microservice_grpc/frontend/service.bal",
                 "startPosition": {
-                  "line": 212,
+                  "line": 213,
                   "offset": 38
                 },
                 "endPosition": {
-                  "line": 212,
+                  "line": 213,
                   "offset": 87
                 }
               },
               "diagnostics": []
             }
           ],
           "returns": [
-            "ballerina/http:http:2.6.0:Ok",
-            "ballerina/http:http:2.6.0:Unauthorized",
+            ":Ok",
+            ":Unauthorized",
```

Comment:
@AzeemMuzammil is this correct? it seems we are missing package information now

Reply:
Probably that's because of the new changes done to the HTTP package, will check

Reply:
Any updates?

Reply:
There is no code change required, It's just a test resource issue, Basically when tests are running `http:$version` part is trimmed off, that's the reason why there is a diff, Actually code generates without an issue.

Reply:
Ack
@NipunaRanasinghe Can u approve as well?

Reply:
Approved